maintainers/maintainer-list.nix

```diff
@@ -2450,6 +2450,12 @@
     githubId = 89596;
     name = "Florian Friesdorf";
   };
+  ChaosAttractor = {
+    email = "lostattractor@gmail.com";
+    github = "LostAttractor";
+    githubId = 46527539;
+    name = "ChaosAttractor";
+  };
```

Comment:
Don't need this right now, and you should squash the commits into one: `kuro: init at 8.1.8`